doc/cephfs/createfs.rst

```diff
@@ -77,6 +77,12 @@ If you have created more than one file system, and a client does not
 specify a file system when mounting, you can control which file system
 they will see by using the `ceph fs set-default` command.
 
+Adding a Data Pool to the MDS
```

Comment:
```suggestion
Adding a Data Pool to the file system
```

Can you fix it here and in the other section.

Reply:
I've made this change (s/MDS/File System/) in the titles. I left the sentence intact in the targeted section, but I'll change it if it's your judgement that it should be changed.

Thanks for the quick turnaround on this review, Patrick.